### PR TITLE
API project files

### PR DESCRIPTION
**Pull request by Schnee18**
First draft of GET, POST, and README files for the API final project.

### CODE REVIEW

File: student-work/2023/Sharon-Schnee/api-final-project/post.md [hunk 1]
```diff
@@ -1 +1,112 @@
...
+
+Include the authorization header using the following structure. For more information about Authentication, see the [ReadMe](https://github.com/Schnee18/student-showcase/blob/main/student-work/2023/Sharon-Schnee/api-final-project/readme.md) file.
+
+```
+curl --request GET https://api.gphd.net/lunch/burgermeal
+--header 'Content-Type: application/json'
+--header 'Authorization: Bearer ImahFWLZWFdD8VVcUtIED2YuOjPFlZpldQTE5tUqKdv'
+```
+
+```
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sharon-Schnee/api-final-project/post.md:59]:
```suggestion
``` JSON
```

File: student-work/2023/Sharon-Schnee/api-final-project/readme.md [hunk 2]
```diff
@@ -1 +1,78 @@
+# Hamburger POC API Guide
+
+1. [Getting started](#getting-started)
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sharon-Schnee/api-final-project/readme.md:3]:
```suggestion
[Getting started](#getting-started)
```

File: student-work/2023/Sharon-Schnee/api-final-project/readme.md [hunk 3]
```diff
@@ -1 +1,78 @@
...
+2. [Authentication](#authentication)
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sharon-Schnee/api-final-project/readme.md:4]:
```suggestion
[Authentication](#authentication)
```

File: student-work/2023/Sharon-Schnee/api-final-project/readme.md [hunk 4]
```diff
@@ -1 +1,78 @@
...
+3. [API reference](#API-reference)
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sharon-Schnee/api-final-project/readme.md:5]:
```suggestion
[API reference](#API-reference)
```

File: student-work/2023/Sharon-Schnee/api-final-project/readme.md [hunk 5]
```diff
@@ -1 +1,78 @@
+# Hamburger POC API Guide
+
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sharon-Schnee/api-final-project/readme.md:2]: ## Additional Information

Comment (Laura-Novich-OBW) [reply student-work/2023/Sharon-Schnee/api-final-project/readme.md:2]: @Schnee18 - consider moving this to the end of the page

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
small changes but otherwise looking good!

File: student-work/2023/Sharon-Schnee/api-final-project/readme.md (hunk 5 above)
Comment (Schnee18) [reply student-work/2023/Sharon-Schnee/api-final-project/readme.md:2]: Thanks for the feedback! Two questions:

1. What should I move to the end of the page? The heading "Hamburger POC API Guide"? 
2. Also for the links to different parts of the readme, I wanted it in a list. Did you just want the numbers taken out, or should it not be in a list?

File: student-work/2023/Sharon-Schnee/api-final-project/post.md [hunk 6]
```diff
@@ -1 +1,112 @@
...
+---- | ----- | ----- | -------------------- | ---
+type | string | N |  Choice of side. Options include frenchFries, onionRings, sweetPotatoFries, spicyFries, coleSlaw, sideSalad, and none. | frenchFries |
+size | string | N |Size of side order. Options include medium and large. | medium |
+
+#### Drink object
+Name | type | Required | Description | Default 
+---- | ----- | ----- | -------------------- | ---
+type | string  | N | Type of drink. Options include coke, dietCoke, sprite, iceTea, orangeJuice, appleJuice, mangoNectar, peachNectar, and none | coke |
+size | string | N |  Size of drink. Options include small, medium, large, and xLarge. | medium |
+drink | boolean  | N | With or without ice. Options include true=with ice, false=without ice. | true |
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sharon-Schnee/api-final-project/post.md:42]:
```suggestion
ice | boolean  | N | With or without ice. Options include true=with ice, false=without ice. | true |
```

File: student-work/2023/Sharon-Schnee/api-final-project/post.md [hunk 7]
```diff
@@ -1 +1,112 @@
...
+## Examples
+
+### Request
+
+Include the authorization header using the following structure. For more information about Authentication, see the [ReadMe](https://github.com/Schnee18/student-showcase/blob/main/student-work/2023/Sharon-Schnee/api-final-project/readme.md) file.
+
+```
+curl --request GET https://api.gphd.net/lunch/burgermeal
+--header 'Content-Type: application/json'
+--header 'Authorization: Bearer ImahFWLZWFdD8VVcUtIED2YuOjPFlZpldQTE5tUqKdv'
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sharon-Schnee/api-final-project/post.md:56]: Lines 55 and 56 are examples for authorization (using HTTP headers) so if you want to include it in that section you can.

File: student-work/2023/Sharon-Schnee/api-final-project/readme.md (hunk 5 above)
Comment (Laura-Novich-OBW) [reply student-work/2023/Sharon-Schnee/api-final-project/readme.md:2]: the links to additional information should be moved to the end of the page

**Review by Laura-Novich-OBW — APPROVED** [review]

**Pull request merged**